entity-service-api/src/main/proto/org/hypertrace/entity/change/event/v1/entity_change_event_value.proto

```diff
@@ -0,0 +1,20 @@
+syntax = "proto3";
+
+option java_multiple_files = true;
+
+package org.hypertrace.entity.data.service.v1;
+
+import "org/hypertrace/entity/data/service/v1/entity.proto";
+
+message EntityChangeEventValue {
+  EventType event_type = 1;
+  optional Entity previous_version = 2; // previous will not be present for newly created entity
+  optional Entity latest_version = 3; // latest version will not be present for delete entity
+
+  enum EventType {
```

Comment:
up to you - can probably replace the enum with a oneof containing three specific messages  (this just represents the comments you wrote directly in the API spec rather than in comments). We don't plan any future changes (famous last words) here, but it also would mean if we added another event type we need not make it match with these other fields.